### PR TITLE
EZAF-6868 Updated ResourceQuotas in Spark Examples

### PR DESCRIPTION
**Pull request by poral-yuktha**
- Updated Spark image to gcr.io/mapr-252711/spark-3.5.1:v3.5.1.0.4
- Updated ResourceQuotas under spec in Spark Examples
- Tested the same in Apps Cluster with 1.6 Build 
image (2)

### CODE REVIEW

File: Model-Monitoring/Spark/Data_Validation_Spark.ipynb [hunk 1]
```diff
@@ -174,7 +174,7 @@
    ],
    "source": [
     "from pyspark import SparkFiles\n",
-    "spark_dataframe = spark.read.format('parquet').load(\"file:///mounts/shared-volume/shared/airbnb_listings.parquet\")"
+    "spark_dataframe = spark.read.format('parquet').load(\"file:///mounts/shared-volume/shared/ezua-tutorials/current-release/Model-Monitoring/Spark/airbnb_listings.parquet\")"
```
Comment (idzikovskyi) [inline Model-Monitoring/Spark/Data_Validation_Spark.ipynb:177]: Should we include "current-release" path section here?

**Review by idzikovskyi — APPROVED** [review]

File: Model-Monitoring/Spark/Data_Validation_Spark.ipynb (hunk 1 above)
Comment (poral-yuktha) [reply Model-Monitoring/Spark/Data_Validation_Spark.ipynb:177]: Dataset is actually present in that path

**Comment by poral-yuktha** [comment]
Tested Spark-GPU examples
image (3)

**Review by skandtandon — APPROVED** [review]

**Review by skandtandon — APPROVED** [review]

File: Model-Monitoring/Spark/Whylogs_Profiling_Spark.ipynb [hunk 2]
```diff
@@ -18,7 +18,7 @@
     "\n",
     "- Select Add Endpoint\n",
     "- Select Single Sign-On\n",
-    "- Select Create Session, selecting language as Python and in properties use \"spark.kubernetes.container.image\": \"gcr.io/mapr-252711/spark-whylogs-3.5.1:v3.5.1.0.2\" the whylogs -integerated spark image in properties.\n",
+    "- Select Create Session, selecting language as Python and in properties use \"spark.kubernetes.container.image\": \"gcr.io/mapr-252711/spark-3.5.1:v3.5.1.0.5\" the whylogs -integerated spark image in properties.\n",
```
Comment (Venkata-Manideep) [inline Model-Monitoring/Spark/Whylogs_Profiling_Spark.ipynb:21]: revert the image to earlier whylog image.

File: Model-Monitoring/Spark/Data_Validation_Spark.ipynb (hunk 1 above)
Comment (Venkata-Manideep) [reply Model-Monitoring/Spark/Data_Validation_Spark.ipynb:177]: same here,revert to earlier whylog image

**Review by ask664 — APPROVED** [review]

**Comment by ask664** [comment]
@poral-yuktha  and @skandtandon  please resolve conflict.

**Pull request merged**